index.css

```diff
@@ -0,0 +1,79 @@
+@import url('https://fonts.googleapis.com/css2?family=Poppins&display=swap');
+
+#container { display: flex; }
+
+nav {
+  padding-top: 10px;
+}
+
+@media only screen and (max-width: 600px) {
+  nav {
+    display: flex;
+    justify-content: space-between;
+    top: 0;
+    width: 100%;
+    position: fixed;
+  }
```

Comment:
- [ ] There is a bigger space around the elements of the navbar, we could use padding to be similar to the Figma template 👇🏽 

| Figma template | Your design |
|---|---|
| ![image](https://user-images.githubusercontent.com/58642949/204933679-1da86da1-f073-4beb-837f-912b98e94f45.png) | ![image](https://user-images.githubusercontent.com/58642949/204933776-540bd535-af63-4222-b42c-548f07ae99b4.png) |